vmdb/lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -131,6 +131,7 @@ class AutomateMethodException < StandardError
   Socket.do_not_reverse_lookup = true  # turn off reverse DNS resolution
 
   require 'drb'
+  require 'yaml'
```

Comment:
@matthewd was this expected?  I haven't looked, do you have any idea what changed in activesupport that requiring activesupport/all or something else that doesn't require yaml any longer?